_posts/2023-03-22-cockpit-288.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Cockpit 288
+author: kkoukiou
+date: '2023-03-22'
+tags: cockpit, machines, podman
+slug: cockpit-288
+category: release
+summary: 'Metrics redesign, accounts page improvements, virtual machines cloud base editing, services page speed ups, chronyd manual time servers, esbuild bundler'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 288, cockpit-machines 286, and cockpit-podman 65:
+
+## Accounts: Show shell and home directory on detail page
+
+Account detail pages show the home directory and shell for an account.
+
+![details page for an account, showing the newly added shell and home directory entries]({{ site.baseurl }}/images/288-accounts-show-shell-and-home-directory.png)
+
+## Accounts: Custom user ID during account creation
+
+A custom user ID can now be specified during account creation. It defaults to the next available user ID on the system.
+
+![account creation dialog showing an entry for user ID]({{ site.baseurl }}/images/288-accounts-custom-user-id-during-account-creation.png)
+
+## Overview: Support additional timeservers with chronyd
+
+Cockpit can now configure additional time servers when chronyd is used to synchronize the clock. Previously, this was only supported with systemd-timesyncd.
+
+![screenshot of support for additional timeservers with chronyd]({{ site.baseurl }}/images/288-overview-support-for-additional-timeservers-with-chronyd.png)
+
+## Metrics: Show longer time span by default
+
+The PCP metrics graphs have been redesigned to only show a summary of events for each hour, so that more data fits on the page. The hours can be expanded to get more details.
+
+![screenshot of new design with more info in default view]({{ site.baseurl }}/images/288-metrics-new-design-with-more-info-in-default-view.png)
+
+## Storage: Mounting filesystems at boot time
+
+Filesystems created in Cockpit are usually not used for the main operating system, so they are often supplemental. As such, they now default to `nofail`, to allow a server to boot even if the additional filesystem is not mounted.
+
+The new "at boot" option provides several choices with explanations of when the filesystem would be mounted and what would happen if it was not mounted.
+
+![screenshot of the "nofail" and "_netdev" options are exposed as a dropdown]({{ site.baseurl }}/images/288-storage-the-nofail-and-netdev-options-are-exposed-as-a-dropdown.png)
+
+## Machines: Create VM based on cloud image and start it later
+
+A VM created from a "cloud base" image can now use the "Create and edit" mode. This is a way to edit all virtual machine properties (bootloader, boot order, disks, networking, etc.) before the VM is installed.
+
+## Services: Units need to be re-pinned
```

Comment:
Wait this was part of the previous release. Just forgotten release note. https://github.com/cockpit-project/cockpit/pull/18416